docs/docs/lib/java.mdx

```diff
@@ -0,0 +1,192 @@
+| `isQaMode`            | `Boolean`              | Whether the SDK is in QA mode. Not for production use. If true, random assignment is disabled and only explicitly forced variations are used (default: `false`)                                                                                                                                                                                                |
+| `url`                 | `String`               | The URL of the current page. Useful when evaluating features based on the page URL.                                                                                                                                                                                                                                                                            |
+| `forcedVariationsMap` | `Map<String, Integer>` | Force specific experiments to always assign a specific variation (used for QA)                                                                                                                                                                                                                                                                                 |
+| `trackingCallback`    | `TrackingCallback`     | A callback that will be invoked with every experiment evaluation where the user **is** included in the experiment. See [TrackingCallback](#tracking-callback). To subscribe to all evaluated events regardless of whether the user is in the experiment, see [Subscribing to experiment runs](#subscribing-to-experiment-runs-with-the-experimentruncallback). |
+
+#### Using the GBContext builder
+
+The builder is the easiest to use way to construct a `GBContext`, allowing you to provide as many or few arguments as you'd like. All fields mentioned above are available via the builder.
+
+<CodeTabsGBContextBuilderPartial />
+
+#### Using the GBContext constructor
+
+You can also use `GBContext` constructor if you prefer, which will require you to pass all arguments explicitly.
+
+<CodeTabsGBContextConstructorPartial />
+
+For complete examples, see the [Examples](#examples) section below.
+
+#### Features
+
+The features JSON is equivalent to the `features` property that is returned from the SDK Endpoint for the specified environment. You can find your endpoint on the [Settings &rarr; API Keys page <ExternalLink/>](https://app.growthbook.io/settings/keys).
+
+- You can read more [about features here](/app/features)
+- You can see an [example features JSON here <ExternalLink/>](https://github.com/growthbook/examples/blob/main/jvm-spring-web/src/main/resources/static/features.json)
```

Comment:
We also have example JSON at https://docs.growthbook.io/app/api#the-sdk-endpoint .  Might make more sense to keep people on the docs instead of linking off to GitHub when possible.